Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM node:17-bullseye
+FROM denoland/deno
 WORKDIR /app
-CMD wget -qO- https://raw.githubusercontent.com/xorgram/xor/main/scripts/start.sh | sh
+CMD deno run --allow-env --allow-net --allow-read --allow-run --allow-write --import-map=https://raw.githubusercontent.com/xorgram/xor/deno/import_map.json -r https://raw.githubusercontent.com/xorgram/xor/deno/main.ts
```

Comment:
using `-A` is better IMO.

Reply:
Why?

Reply:
We won't need more permissions. The remaining permissions `-A` will add will be `ffi` and `hrtime`, which are very unlikely to get used. And even if we do, yes, we will update the Dockerfile. It's not secure, nor a good practice to do `-A` in production, and I've not seen anyone do it.